pkg/maps/policymap/policymap.go

```diff
@@ -117,6 +117,11 @@ func (k *PolicyKey) GetDestPort() uint16 {
 	return byteorder.NetworkToHost16(k.DestPortNetwork)
 }
 
+// GetPortMask returns the port mask of the key
+func (k *PolicyKey) GetPortMask() uint16 {
+	return 0xffff
```

Comment:
Should this be conditional on `DestPortNetwork` being 0?

Reply:
Eventually. The pull request that finally introduces port ranges will add logic here.